README.md

```diff
@@ -58,6 +78,10 @@
 - [AutoConfig](https://github.com/shedaniel/AutoConfig) - A full-fledged, annotation-based configuration library. ([Wiki](https://shedaniel.gitbook.io/cloth-config/auto-config)) `Apache-2.0`
 - [Cloth Config](https://github.com/shedaniel/cloth-config) - A client side API for creating configuration screens. ([Wiki](https://shedaniel.gitbook.io/cloth-config)) `Unlicense`
 
+### Data Parsing & Loading
+
+- [Bow Tie](https://github.com/Siphalor/bow-tie/) - Data loading interceptor that enables the use of Hjson, YAML, XML and more instead of JSON. `APACHE`
```

Comment:
Not your fault, but I just noticed, that I used the wrong license identifier for this.
Should be `Apache-2.0` instead of `APACHE`.

You don't *need* to do anything about this, I'll fix it when this is merged :)